tests/basic/storage_pool_test.go

```diff
@@ -5794,3 +5794,226 @@ func addNewPools(n node.Node, numPools int) error {
 	}
 	return nil
 }
+
+var _ = Describe("{ChangedIOPriorityPersistPoolExpand}", func() {
+	var testrailID = 79487
+	// Testrail Description : Changed pool IO_priority should persist post pool expand
+	// Testrail Corresponds : https://portworx.testrail.net/index.php?/cases/view/79487
+	var runID int
+
+	JustBeforeEach(func() {
+		StartTorpedoTest("ChangedIOPriorityPersistPoolExpand",
+			"Changed pool IO_priority should persist post pool expand",
+			nil, testrailID)
+		runID = testrailuttils.AddRunsToMilestone(testrailID)
+	})
+
+	var contexts []*scheduler.Context
+	stepLog := "Changed pool IO_priority should persist post pool expand"
+	It(stepLog, func() {
+		log.InfoD(stepLog)
+
+		contexts = make([]*scheduler.Context, 0)
+		for i := 0; i < Inst().GlobalScaleFactor; i++ {
+			contexts = append(contexts, ScheduleApplications(fmt.Sprintf("changedioprioritypoolexpand-%d", i))...)
+		}
+		ValidateApplications(contexts)
+		defer appsValidateAndDestroy(contexts)
+
+		// Get the Pool UUID on which IO is running
+		poolUUID, err := GetPoolIDWithIOs()
+		log.InfoD("Pool UUID on which IO is running [%s]", poolUUID)
+		log.FailOnError(err, "Failed to get pool using UUID [%v]", poolUUID)
```

Comment:
If `GetPoolIDWithIOs()` fails, it won't return `poolUUID`, and always `log.FailOnError()` first before doing `log.InfoD()`